src/styles/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    background-size: cover;
+    background-image: url("../images/imac.jpeg");
+    margin-bottom: 40px;
+  }
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+  &__description {
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-color;
+  }
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    &-count {
+      padding-top: 2px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    &-title {
+      display: block;
+      text-transform: capitalize;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $second-color;
+    }
+    &-amount {
+      display: block;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+  &__btn {
+    display: block;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    padding: 12px 70px;
+    border-radius: 5px;
+    background-color: #00acdc;
```

Comment:
also make variable for this color and try to set up right height for button, i think it will fix some test